### PR TITLE
FIX: typo in func initialization for reduction

### PR DESCRIPTION
**Pull request by samir-nasibli**

### CODE REVIEW

File: dpnp/backend/kernels/dpnp_krnl_reduction.cpp [hunk 1]
```diff
@@ -143,7 +143,7 @@ void func_map_init_reduction(func_map_t& fmap)
     // WARNING. The meaning of the fmap is changed. Second argument represents RESULT_TYPE for this function
     // handle "out" and "type" parameters require user selection of return type
     // TODO. required refactoring of fmap to some kernelSelector
-    fmap[DPNPFuncName::DPNP_FN_SUM][eft_INT][eft_INT] = {eft_LNG, (void*)dpnp_sum_c<int, int>};
+    fmap[DPNPFuncName::DPNP_FN_SUM][eft_INT][eft_INT] = {eft_INT, (void*)dpnp_sum_c<int, int>};
```
Comment (densmirn) [inline dpnp/backend/kernels/dpnp_krnl_reduction.cpp:146]: https://github.com/IntelPython/dpnp/pull/603#discussion_r576712941

Comment (samir-nasibli) [reply dpnp/backend/kernels/dpnp_krnl_reduction.cpp:146]: This is little bit confused.

@densmirn This is bad practice doing smth for python level requirements.
We have only instances for ints here.

@shssf Currently I am working on global solution for `fmap`. Let's leave this PR as is for now.
I will update it or close in future dependently on the solution.

**Comment by shssf** [comment]
tests should not pass because the original code is correct. Changed code is wrong.

**Comment by samir-nasibli** [comment]
> tests should not pass because the original code is correct. Changed code is wrong.

@shssf the original code is correct for the tests but not clear. Provided solution just doesn't fix it. Also see, https://github.com/IntelPython/dpnp/pull/610#discussion_r579105184

**Pull request closed without merging**